### ISSUE TITLE
Wayland support

### ISSUE THREAD

**Issue by antistress** [flathub/org.frozen_bubble.frozen-bubble#6]
Hi,
Sorry it's more a question than a bug report.
I've run org.frozen_bubble.frozen-bubble on my Debian GNOME-Wayland system, and looking at $ xlsclients it seems to run natively on Wayland (without XWayland)
Which seems strange to me since I believe that Frozen Bubble uses SDL 1.2 which is not Wayland compatible and that I don't see any Wayland socket in /var/lib/flatpak/exports/share/applications/org.frozen_bubble.frozen-bubble.desktop
So : what is the magic ? Does org.frozen_bubble.frozen-bubble run natively under Wayland or not ?
Thanks !

**Comment by AsciiWolf** [comment]
Hmm, that's weird, Frozen Bubble should not run natively on Wayland. You can see [here](https://github.com/flathub/org.frozen_bubble.frozen-bubble/blob/master/org.frozen_bubble.frozen-bubble.json#L10) that there is no "wayland" socket in finish-args.

**Comment by antistress** [comment]
So maybe I should fill a bug against Mutter or xlsclients ?

**Comment by AsciiWolf** [comment]
Are you still experiencing this issue?

**Comment by antistress** [comment]
yes I'm still experiencing this issue. 
Indeed, while running Flatpak Frozen Bubble, $ xlsclients prints :
HAL  gsd-xsettings
HAL  ibus-x11
HAL  gnome-shell

But I can complete my report since at present time I'm running "XWayland on demand" experimental feature
That means that running $ xlsclients should print nothing.
But once Flatpak Frozen Bubble is started, $ xlsclients prints :
HAL  gsd-xsettings
HAL  ibus-x11
HAL  gnome-shell

Which seems to indicate that starting Flatpak Frozen Bubble also starts XWayland
However xlsclients doesn't seem to see Flatpak Frozen Bubble

**Comment by antistress** [comment]
Maybe that command is more reliable

Without Flatpak Frozen Bubble running :
$ ps ax|grep Xwayland
 157085 pts/0    S+     0:00 grep Xwayland

With  Flatpak Frozen Bubble running :
$ ps ax|grep Xwayland
 157331 ?        Sl     0:00 /usr/bin/Xwayland :0 -rootless -noreset -accessx -core -auth /run/user/1000/.mutter-Xwaylandauth.G5W1V0 -listen 4 -listen 5 -displayfd 6 -listen 7
 157402 pts/0    S+     0:00 grep Xwayland

**Comment by AsciiWolf** [comment]
That's correct. Flatpak Frozen Bubble does not run natively on Wayland, therefore it uses Xwayland (X11 Server running on Wayland).
